### PR TITLE
Added bash script for BitMLx pipelining

### PR DESCRIPTION
**Pull request by luiminyan**
A bash script is added for the BitMLx pipelining.

### CODE REVIEW

File: BitMLx_pipeline.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+<<COMMENT
+This script is used to pipelining all executions on files stored under "./output".
```
Comment (QPotato) [inline BitMLx_pipeline.sh:3]:
```suggestion
This script is used for pipelining all executions on files stored under "./output".
```

File: BitMLx_pipeline.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    1. Run the Bitmlx compiler
+    2. Replace all hashes with python script
+    3. Compile each modified contract and store result in corresponding .blz files
+
+Usage: 
+    ./BitMLx_pipeline.sh
+COMMENT
+
+stack run || {
+    echo "Error: stack run failed."
```
Comment (QPotato) [inline BitMLx_pipeline.sh:15]:
```suggestion
    echo "Error: BitMLx compiler failed."
```

File: BitMLx_pipeline.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    exit 1
+}
+
+file_ext=".rkt"
+result_ext=".blz"
```
Comment (QPotato) [inline BitMLx_pipeline.sh:20]: For next PR: I just learned that the standard balzac extension is actually ".balzac", so we may want to change this and the python script to use that.

Comment (luiminyan) [reply BitMLx_pipeline.sh:20]: Do you mean to change the result file extension into ".balzac"? I did it in my local branch, by changing the extension in the bash file, and everything is still working as before. It seems that one does not need to change something in the python script.

**Pull request merged**